### DISCUSSION TITLE
After being deleted, tables are still shown after 'Run'

### DISCUSSION THREAD

**Discussion by erakastrati** [Budibase/budibase#2313] [General]
Hi there! 
I hope all of you are doing well. I am recently analyzing  'no code, low code' platforms, and lately I've been working with Budibase. 
I am having a problem with some of the concepts in here. For example, I create a table and then  I recognize that I have made some mistakes on that particular table and I decide to delete it. It is all working great until now, the table has been removed from the menu. But as soon as I publish  the project and run it, the table is still on the menu, even that it doesn't figure on the Design and Data module. How can I delete it from there, is there any way to do this or should I delete the project and rebuild it again?
Thanks for your time,
Era

**Comment by kevmodrome** [comment]
Hi! You need to remove the link from the navigation settings in your Layout. You can find the layouts in the top left part of the design section, next to Screens.

**Reply by erakastrati** [reply]
Thanks a lot Kevin. I didn't recognize it at all. 
I really appreciate it.
